.github/workflows/browserslist-db-update.yml

```diff
@@ -0,0 +1,38 @@
+name: Update Browserslist Database
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-browserslist-database:
+    name: Create PR containing Browserslist updates
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js (.nvmrc)
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'npm'
+
+      - name: Configure git
+        run: |
+          git config --global user.email "actions@github.com"
+          git config --global user.name "GitHub Actions"
+
+      - name: Update Browserslist database and create PR if applicable
+        uses: c2corg/browserslist-update-action@v1
```

Comment:
v1 uses the command that will eventually be deprecated? Is this expected to still work for the foreseeable future?

Reply:
@aaemnnosttv That is correct, v1 uses the legacy command. I couldn't find an official statement regarding how long it will be supported, however, [the action we're using mentions](https://github.com/marketplace/actions/browserslist-update-action#:~:text=v1%20uses%20command%20npx%20browserslist%40latest%20%2D%2Dupdate%2Ddb%20which%20is%20now%20deprecated%20and%20to%20be%20removed%20in%20a%20future%20major%20browserslist%20release) that it will be removed in a future major update of Browserslist.

However, what's interesting is that, the update DB package was [only moved to a different package](https://github.com/browserslist/browserslist/commit/7e019ce761242e1eb9777c56bb85853e41593f09) for isolation. The `npx browserslist@latest --update-db` command that v1 of the action uses actually utilizes the new `update-browserslist-db` package under the hood:
https://github.com/browserslist/browserslist/blob/7f98081cba6abe39bba06ba97070f58ebabdbbd8/cli.js#L3

v2 of the action also uses `update-browserslist-db` package, but doesn't work with npm < 6 due to [this check](https://github.com/browserslist/update-db/blob/597c6d12268e9c55368c754269872786662fe4b0/check-npm-version.js) added in [its `cli` script](https://github.com/browserslist/update-db/blob/597c6d12268e9c55368c754269872786662fe4b0/cli.js#L6).

As we discussed before, I opened #6026 to explore updating to a newer LTS of Node. While that is being explored, we can consider pinning the version of Browserslist in our package so that it doesn't update past version `4.x.x`, just to be in the safe side. What do you think?

Reply:
Thank you, @tofumatt! Do you think we need to include `browserslist` in our project with its version pinned to `~4.x.x` so that it doesn't go through major updates in the future, just to be in the safe side?